### DISCUSSION TITLE
how to use blocklist.txt/blacklist.txt

### DISCUSSION THREAD

**Discussion by marcatmadu** [rustdesk/rustdesk-server#309] [General]
I have been searching in the documentation on how to use the blocklist.txt & blacklist.txt (hbbr) but was unable to find something.
Anyone can explain how to use these ?

Thanks !

**Comment by pdifeo** [comment]
I'd like to know about it too

**Reply by marcatmadu** [reply]
I have read somewhere that it should contain a list with IP-addresses to block. I tried that, restarted hbbr but those clients from behind the blocked IP-adresseses were still able to connect to and from each other.
